osal/win32/osal.c

```diff
@@ -48,6 +48,24 @@ static double qpc2usec;
 
 #define USECS_PER_SEC     1000000
 
+int osal_getabsolutetimeofday(struct timeval *tv, struct timezone *tz)
```

Comment:
This function provides a new system time value every 1-15 ms (depending on the settings made with [timeBeginPeriod](https://msdn.microsoft.com/en-us/library/windows/desktop/dd757624(v=vs.85).aspx)). On Windows 8 and above, there is the new function [GetSystemTimePreciseAsFileTime](https://msdn.microsoft.com/en-us/library/windows/desktop/hh706895(v=vs.85).aspx) that provides the system time with much higher precision (< 1 µs). 

My version of osal_getabsolutetimeofday is only a workaround to initialize DC with an absolute time but needs some modifications before publishing. I would prefer to create a separate pull request for this issue. As I wrote below, I mistakenly commited serveral changes to this pull request.